specification/cluster/reroute/ClusterRerouteRequest.ts

```diff
@@ -46,7 +46,7 @@ export interface Request extends RequestBase {
      */
     metric?: Metrics
     /**
-     * If true, then retries allocation of shards that are blocked due to too many subsequent allocation failures.
+     * If true, then retries allocation of shards that are ed due to too many subsequent allocation failures.
```

Comment:
I think this must be a typo
```suggestion
     * If true, then retries allocation of shards that are blocked due to too many subsequent allocation failures.
```